tg_react/management/commands/react_makemessages.py

```diff
@@ -96,6 +87,13 @@ def handle(self, *args, **options):
             'ignore_patterns': [],
             'use_default_ignore_patterns': True,
             'domain': 'django',
+            'no_color': True,
```

Comment:
print out that the user needs to manually run makemessages (and strip that logic from this file)